lix/packages/sdk/src/database/serializeJsonPlugin.ts

```diff
@@ -0,0 +1,97 @@
+import {
```

Comment:
did not check/ understand this file, i assume its some copy paste from kysley somwhere ...

Reply:
I wrote it myself based on a kysely plugin. The API is bonkers. The important part is the `serializeJson` function.